### PR TITLE
C & N | 7 | implement building headers for request to nameserver

### PR DESCRIPTION
**Pull request by chelseakomlo**
Sets headers for a request to a nameserver

### CODE REVIEW

File: src/resolver.h [hunk 1]
```diff
@@ -1 +1,6 @@
+#include "types.h"
+
 char* resolve(char*);
+
+header* build_header();
+void reset_counter();
```
Comment (rosatolen) [inline src/resolver.h:7]: Why does the domain export these functions as part of resolver? Is the idea to allow other modules to build_header() through the resolver and use it for other functions? Similar question for reset_counter.

**Comment by chelseakomlo** [comment]
We could have used strtok, but it was useful writing this by hand.

File: src/resolver.h (hunk 1 above)
Comment (chelseakomlo) [inline src/resolver.h:7]: We use these functions in our test file.

**Comment by chelseakomlo** [comment]
Fixing build!

**Pull request closed without merging**